src/types/KeyPairSigner.ts

```diff
@@ -27,3 +33,11 @@ export interface KeyPairSigner {
 export interface KeyPairSignerOptions {
   readonly data: Uint8Array | Uint8Array[];
 }
+
+/**
+ * Key pair signer for blindSign options
+ */
+export interface KeyPairSignerForBlindSignOptions {
+  readonly data: Uint8Array | Uint8Array[];
+  proverCommitment: Uint8Array;
```

Comment:
```suggestion
  readonly proverCommitment: Uint8Array;
```
How about defining `proverCommitment` as readonly just like the other properties? It seems not necessary to be mutable here.